bitnami/etcd/README.md

```diff
@@ -582,6 +582,27 @@ If you encounter errors when working with persistent volumes, refer to our [trou
 | `serviceAccount.annotations`                  | Additional annotations to be included on the service account | `{}`    |
 | `serviceAccount.labels`                       | Additional labels to be included on the service account      | `{}`    |
 
+### Defragmentation parameters
+
+| Name                                        | Description                                                                                                                                | Value       |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
+| `defrag.enabled`                            | Enable automatic defragmentation. This is most effective when paired with auto compaction: consider setting "autoCompactionRetention > 0". | `false`     |
+| `defrag.cronjob.startingDeadlineSeconds`    | Number of seconds representing the deadline for starting the job if it misses scheduled time for any reason                                | `""`        |
+| `defrag.cronjob.schedule`                   | Schedule in Cron format to defrag (daily at midnight by default)                                                                           | `0 0 * * *` |
+| `defrag.cronjob.concurrencyPolicy`          | Set the cronjob parameter concurrencyPolicy                                                                                                | `Forbid`    |
+| `defrag.cronjob.suspend`                    | Boolean that indicates if the controller must suspend subsequent executions (not applied to already started executions)                    | `false`     |
+| `defrag.cronjob.successfulJobsHistoryLimit` | Number of successful finished jobs to retain                                                                                               | `1`         |
+| `defrag.cronjob.failedJobsHistoryLimit`     | Number of failed finished jobs to retain                                                                                                   | `1`         |
+| `defrag.cronjob.labels`                     | Additional labels to be added to the Defrag cronjob                                                                                        | `{}`        |
+| `defrag.cronjob.annotations`                | Annotations to be added to the Defrag cronjob                                                                                              | `{}`        |
```

Comment:
I'm missing some common parameters that we usually expose for every cronjob, job, deployment, etc.:

- nodeSelector
- tolerations
- podSecurityContext
- containerSecurityContext
- serviceAccountName